node/cn/backend.go

```diff
@@ -499,6 +510,32 @@ func (s *CN) SetComponents(component []interface{}) {
 	// do nothing
 }
 
+func (s *CN) SetupKaiaxModules() error {
+	// Declare modules
+	var (
+		mNoop = noop.NewNoopModule()
+	)
+
+	// Register modules to respective components
+	s.RegisterBaseModules(mNoop)
```

Comment:
Are we going to leave the noop module registered as a template, or will this be removed after introducing other modules?

Reply:
It will be removed after introducing other modules.